src/perl/vdotest/VDOTest.pm

```diff
@@ -226,48 +221,6 @@ sub new {
   return $self;
 }
 
-########################################################################
-# Create a SystemTap command object to invoke on the test machines.
-#
-# @param  script      The systemtap script to be run
-# @oparam stapArgs    A list of additional parameters to pass to systemtap
-# @oparam scriptArgs  Any additional parameters to be passed to the script
-#
-# @return  the new command object
-##
-sub makeSystemTapCommand {
-  my ($self, $script, $stapArgs, @scriptArgs) = assertMinArgs(2, @_);
-  my %args = (
-              machine    => $self->getUserMachine(),
-              outputDir  => $self->{runDir},
-              script     => "$self->{binaryDir}/systemtap/$script",
-              scriptArgs => \@scriptArgs,
-              stapArgs   => $stapArgs // "",
-             );
-  return Permabit::SystemTapCommand->new(%args);
-}
-
-########################################################################
-# Launch a SystemTap script to record the kernel page cache's
-# retention of blocks in our device.
-#
-# @param devMajor  The block major device number
-# @param devMinor  The block minor device number
-##
-sub _monitorPageCacheStats {
-  my ($self, $devMajor, $devMinor) = assertNumArgs(3, @_);
-  if (!$self->{_pageCacheMonitor}) {
-    my $script = "monitor-page-cache.stp";
```

Comment:
This monitor-page-cache.stp script... does it also need to be removed? Does it still exist?

Either way, it's probably worth mentioning in the commit message.that this feature is also being removed.

Reply:
I did a find on the src and I don't see the script.  "monitorPageCacheStats   => 0", so I think the stp file has been removed when we set that to 0. 

We have jira ticket for systemtap not working.  I am not sure if that is the right jira ticket to add.
I will create a new jira ticket for removing systemtap.